examples/MQ/pixelDetector/src/CMakeLists.txt

```diff
@@ -71,6 +72,7 @@ target_link_libraries(${target} PUBLIC
   FairRoot::Gen # FairBoxGenerator
   FairRoot::GeoBase
   FairRoot::ParBase # FairParGenericSet
+  FairRoot::ExGConfig
```

Comment:
The `FairRoot::ExGConfig` target is built conditionally https://github.com/FairRootGroup/FairRoot/blob/453ab6c945bb7438eccb0061888cd9353841d960/examples/CMakeLists.txt#L15-L17
because `FairMCConfigurator` is built conditionally https://github.com/FairRootGroup/FairRoot/blob/453ab6c945bb7438eccb0061888cd9353841d960/fairtools/CMakeLists.txt#L11-L13
which leads to compile errors if those dependencies are not built. Please revisit the code and add the necessary conditions to the `add_subdirectory()` CMake calls for all the example that have a dependency on `FairRoot::ExGConfig` (or handle it differently if you wish).

Reply:
Okay, we decided to require yaml-cpp. So this should be resolved now.